.whitesource

```diff
@@ -1,8 +0,0 @@
-{
-  "checkRunSettings": {
-    "vulnerableCheckRunConclusionLevel": "failure"
-  },
-  "issueSettings": {
-    "minSeverityLevel": "LOW"
-  }
-}
```

Comment:
There is much better source, than this [bot spam PR](https://github.com/eonx-com/easy-monorepo/commit/995f2d2dce0f49e271e7618b6cd9efd087c5e6d4)

See roave/security-advisories:
https://ocramius.github.io/blog/roave-security-advisories-protect-against-composer-packages-with-security-issues/